Package/PackageActions/SetUserToken.cs

```diff
@@ -0,0 +1,45 @@
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+using OpenTap.Authentication;
+using OpenTap.Cli;
+namespace OpenTap.Package
+{
+    /// <summary> Sets the authentication token for use with a given domain.  </summary>
+    [Display("set-auth-token", Group: "package", Description: "Sets the authentication token for a given domain.")]
+    [Browsable(false)]
+    public class SetUserToken : ICliAction
+    {
+        /// <summary>  Gets or sets the domain. </summary>
+        [UnnamedCommandLineArgument("Domain")]
+        public string Domain { get; set; }
```

Comment:
Rename this to repository and copy-paste the description / flags from the other package actions